ironfish/src/strategy.test.slow.ts

```diff
@@ -274,11 +279,18 @@ describe('Demonstrate the Sapling API', () => {
       transaction.spend(note, witness)
       receiverNote.returnReference()
 
-      const noteForSpender = new NativeNote(spenderKey.public_address, BigInt(10), '')
+      const receiverAddress = generateKeyFromPrivateKey(receiverKey.spending_key).public_address
```

Comment:
```suggestion
      const receiverAddress = receiverKey.public_address
```

Oh I didn't see at first that this was existing code, well, that's weird. Must be some legacy thing we don't need to do anymore.